ocean_lib/example_config.py

```diff
@@ -86,6 +90,11 @@ def get_config_dict(chain_id: int, network_url: str) -> dict:
 
     if chain_id != 8996:
         config_helper["METADATA_CACHE_URI"] = METADATA_CACHE_URI
+        config_helper["ADDRESS_FILE"] = (
```

Comment:
this shouldn't be in Example Config, it should be in `get_contracts_addresses`, similar to how we configure chain_id automatically when undefined.